### ISSUE TITLE
"Click to show javascript error" showing on generated map

### ISSUE THREAD

**Issue by iacisme** [jupyter-widgets/ipyleaflet#1079]
Hi;

Can I please get some help resolving this issue?

The following code generates the map image posted below:

```
import leafmap

center_location = [55.1381, 14.9249]

base_map = leafmap.Map(center = center_location,
                                           zoom = 6,
                                           height = 1000,
                                          #width = 1500,
                                           draw_control = True,
                                           meausre_control = True,
                                           fullscreen_control = True,
                                           attribution_control = True,
                                          )

base_map
```
![image](https://user-images.githubusercontent.com/72644814/206340098-95e031c9-c046-4a6a-a91b-6022ba08ac34.png)

When I click on the error, this is the expanded message:

![image](https://user-images.githubusercontent.com/72644814/206340239-721c0ea1-a4f9-4203-bd3a-60ab28f99dc4.png)

It tells me to look at the console output. Here is a top and bottom clip of the console output:

![image](https://user-images.githubusercontent.com/72644814/206340449-ef54986f-8150-442d-a10e-2b375897de10.png)

![image](https://user-images.githubusercontent.com/72644814/206340569-2f722b85-9360-4c30-ae97-60395057db8d.png)

I've read the troubleshooting guide and tried the recommended steps found there but no success.

Until recently, the installation was working. The only change I've made was upgrade to the latest version of leafmap and leaflet, to try and resolve another issue, but cannot confirm if this is root cause.

leafmap version = 0.14.0
ipleaflet version = 0.17.1
ipywidgets version = 7.7.1
Jupyterlab version = 3.4.6

Thank you!

**Comment by iacisme** [comment]
Sounds good thank you. 

Is there any order to the installation (ie: geemap first, then ipyleaflet, then leafmap) or does it matter?

Also, are there any packages that I should avoid installing in this environment? I just created a new python3.9 virtual environment dedicated to geo-analytics, hopefully to try and minimize future issues.

Right now the only things I have installed are `scipy` and `jupyter lab`. 

Thanks for your help.

--Igor
